### DISCUSSION TITLE
新人自我介绍来啦！

### DISCUSSION THREAD

**Discussion by jodie-cyber** [alc-beijing/SIG#7] [通用论坛]
Hello大家好，我是Jodie，一名在校大学生。我的专业并不是计算机，但是我对开源社区文化一直很感兴趣，因此报名了这次志愿者活动。我的特长是运营及组织活动。
我理解的开源：开放，合作，共同进步
我希望达成的目标：我希望能够在这次活动中发挥自己的特长，和大家各取所长共同进步，并且和大家一起把开源文化在中国发扬光大，让更多人认识拥抱开源！谢谢！

**Comment by WillemJiang** [comment]
欢迎欢迎！ALC Beijing后续会组织很多活动，例如ApacheCon Asia 2021，  还有一些线下的活动。 欢迎成为ALC Beijing志愿者，参与我们的活动！
